introduction.md

```diff
@@ -3,4 +3,5 @@ My favorite Code School path is JavaScript.
 Deadly skills: 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?